go/chat/msgchecker/boxed_checker.go

```diff
@@ -59,6 +59,9 @@ func checkMessageBoxedLength(msg chat1.MessageBoxed) error {
 	case chat1.MessageType_SENDPAYMENT:
 		return boxedFieldLengthChecker("SENDPAYMENT message", len(msg.BodyCiphertext.E),
 			BoxedSendPaymentMessageBodyMaxLength)
+	case chat1.MessageType_REQUESTPAYMENT:
```

Comment:
you're going to have to vendor chat/msgchecker into keybase/keybase/go so gregord can handle these messages